templates/_templating_scripting.py

```diff
@@ -0,0 +1,129 @@
+"""Commands and scripts for administrating templating of files across SciTools repos.
+"""
+#!/usr/bin/env python3
+
+import argparse
+import json
+from pathlib import Path
+import shlex
+from subprocess import CalledProcessError, check_output, run
+from tempfile import NamedTemporaryFile
+from typing import NamedTuple
+
+
+def git_command(command: str) -> str:
+    command = shlex.split(f"git {command}")
+    return check_output(command).decode("utf-8").strip()
+
+
+GIT_ROOT = Path(git_command("rev-parse --show-toplevel")).resolve()
+TEMPLATES_DIR = GIT_ROOT / "templates"
+assert TEMPLATES_DIR.is_dir()
+
+
+class Config:
+    """Convenience to give the config JSON some readable structure."""
+    class Templatee(NamedTuple):
```

Comment:
A very minor preference, but I have to do a bit of mental juggling to figure out which one "Templatee" is. Perhaps renaming it to something like "TargetRepo"? Maybe it's just a me problem, like remembering left and right.